File: 2023/Linguagem_de_Programacao_Multiplataforma/10- Outubro/10/demoAPI2/src/main/java/com/example/demoAPI2/model/Aluno.java
package com.example.demoAPI2.model;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;

@Entity
@Table (name = "Aluno")
public class Aluno {

    @Id
    private int ra;

    @Column
    private String nome;

    public int getRa() {
        return ra;
    }

    public void setRa(int ra) {
        this.ra = ra;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }
}

File: 2024/Desenvolvimento_para_Dispositivos_Moveis/8- Agosto/14/flutter_application_list_view/lib/aluno.dart
// ignore_for_file: prefer_initializing_formals

class Aluno{
  String name="";
  int ra=0;

  String get getName => name;

  set setName(String name) => this.name = name;

  get getRa => ra;

  set setRa( ra) => this.ra = ra;

  Aluno(String name, int ra){
    this.name = name;
    this.ra = ra;
  }
}